src/extensions/filter-control/utils.js

```diff
@@ -73,56 +93,54 @@ export function sortSelectControl (selectControl, orderBy) {
   $selectControl.append($opts)
 }
 
-export function fixHeaderCSS ({ $tableHeader }) {
-  $tableHeader.css('height', '89px')
+export function fixHeaderCSS ({ $tableHeader }, pixels = '89px') {
+  if ($tableHeader.hasClass('table-sm') && pixels === '89px') {
+    $tableHeader.css('height', '49px')
+  }
+  $tableHeader.css('height', pixels)
 }
 
 export function getElementClass ($element) {
   return $element.attr('class').replace('form-control', '').replace('focus-temp', '').replace('search-input', '').trim()
 }
 
 export function getCursorPosition (el) {
-  if (Utils.isIEBrowser()) {
-    if ($(el).is('input[type=text]')) {
-      let pos = 0
-
-      if ('selectionStart' in el) {
-        pos = el.selectionStart
-      } else if ('selection' in document) {
-        el.focus()
-        const Sel = document.selection.createRange()
-        const SelLength = document.selection.createRange().text.length
-
-        Sel.moveStart('character', -el.value.length)
-        pos = Sel.text.length - SelLength
-      }
-      return pos
+  if ($(el).is('input[type=search]')) {
```

Comment:
Why the isIEBrowser check was removed ?
Is this now needed on every browser ?
If not then please add the check  again or if we drop IE support (see other comment) the whole method can be removed.

Reply:
I'm removing IE support. @wenzhixin  please confirm